swift/ql/test/library-tests/dataflow/dataflow/FlowConfig.qll

```diff
@@ -14,10 +15,17 @@ class TestConfiguration extends DataFlow::Configuration {
 
   override predicate isSink(DataFlow::Node sink) {
     exists(CallExpr sinkCall |
-      sinkCall.getStaticTarget().getName() = "sink(arg:)" and
+      sinkCall.getStaticTarget().getName() = ["sink(arg:)", "sink(opt:)"] and
       sinkCall.getAnArgument().getExpr() = sink.asExpr()
     )
   }
 
   override int explorationLimit() { result = 100 }
 }
+
+private class TestSummaries extends SummaryModelCsv {
+  override predicate row(string row) {
+    // dumb model to allow testing flow through optional chaining (`x?.signum()`)
```

Comment:
```suggestion
    // model to allow data flow through `signum()` as though it were an identity function, for the benefit of testing flow through optional chaining (`x?.`).
```
I'm not sure `signum` is the right thing to use in this test as this would really be taint flow, not data flow.  I can see that this is difficult to test any other way though.

Reply:
Yes, I struggled to find an `Int` method that was value-preserving. I could've used a different type too, but it started to look like too many changes for testing such a little thing.